<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCobranca</class>
 <widget class="QWidget" name="frmCobranca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cobrança de Clientes Whatsapp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>toque.ico</normaloff>toque.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Upload de arquivo de clientes</string>
   </property>
   <widget class="QPushButton" name="btnProcurar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Procurar arquivo CSV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Arquivo de extensão .CSV</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Envio de mensagem de cobrança de clientes</string>
   </property>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIAR ENVIO DE MENSAGENS NO WHATSAPP</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>jardimParaiso.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
